LoopBackTests/LBPersistedModelSubclassingTests.m

```diff
@@ -16,8 +16,12 @@
 @interface Widget : LBPersistedModel
 
 @property (nonatomic, copy) NSString *name;
+// Number can be accessed via both NSNumber and long
```

Comment:
I guess this comment might be misleading.  I'll change those to something like the following:
// a Number property can be accessed via either of NSNumber or the primitive type long.